src/game/Player.cpp

```diff
@@ -2218,10 +2218,10 @@ void Player::GiveXP(uint32 xp, Unit* victim)
     if (!isAlive())
         return;
 
-    uint32 level = getLevel();
+    uint8 pLevel = getLevel();
```

Comment:
You don't need to change that. We don't use uint8 in such cases, because CPUs are fastest at operating on integers of their native word size (and it doesn't make any difference in memory usage here anyway). Additionally, you should _only_ use the "p" prefix if you are dealing with pointers. "level" is still the best variable name in this case.